ocaml/xapi/helpers.ml

```diff
@@ -2015,6 +2015,18 @@ let with_temp_out_ch_of_temp_file ?mode prefix suffix f =
   let@ path, channel = with_temp_file ?mode prefix suffix in
   f (path, channel |> with_temp_out_ch)
 
+let make_external_host_verified_rpc ~__context ext_host_address ext_host_cert
+    xml =
+  let@ temp_file, temp_out_ch =
+    with_temp_file "remote-pool-certificate" ".pem"
```

Comment:
Would suggest `remote-pool-certificate` -> `external-host-cert` for consistency.